lib/Widget/dashboard/dashboardGraph.dart

```diff
@@ -1,75 +1,75 @@
-      point: 2,
-      barColor: charts.ColorUtil.fromDartColor(Colors.lightBlue),
-    ),
-    SubscriberSeries(
-      topic: "การอาบน้ำ",
-      point: 0,
-      barColor: charts.ColorUtil.fromDartColor(Colors.pinkAccent),
-    ),
-    SubscriberSeries(
-      topic: "การใช้ห้องน้ำ",
-      point: 1,
-      barColor: charts.ColorUtil.fromDartColor(Colors.pinkAccent),
-    ),
-    SubscriberSeries(
-      topic: "การกลั้นอุจจาระในระยะ\n1 สัปดาห์ที่ผ่านมา\n",
-      point: 3,
-      barColor: charts.ColorUtil.fromDartColor(Colors.pinkAccent),
-    ),
-    SubscriberSeries(
-      topic: "การกลั้นปัสสาวะในระยะ\n1 สัปดาห์ที่ผ่านมา\n",
-      point: 2,
-      barColor: charts.ColorUtil.fromDartColor(Colors.pinkAccent),
-    ),
-    SubscriberSeries(
-      topic: "การล้างหน้า หวีผม แปรงฟัน",
-      point: 3,
-      barColor: charts.ColorUtil.fromDartColor(Colors.pinkAccent),
-    )
-  ];
-  @override
-  Widget build(BuildContext context) {
+  Widget _preopdashboard(BuildContext context) {
     return Container(
-      child: ListView(
-        shrinkWrap: true,
-        children: <Widget>[
+      child: Column(
+        children: [
+          Padding(
+            padding: const EdgeInsets.fromLTRB(10, 0, 10, 10),
+            child: Container(
+              child: Card(
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.stretch,
+                  children: <Widget>[
+                    Container(
+                      margin: EdgeInsets.all(10),
+                      child: Text('พัฒนาการในแต่ละหัวข้อ',
+                          style: TextStyle(
+                              fontSize: 18, color: Color(0xFFC37447))),
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.fromLTRB(50, 0, 50, 20),
+                      child: ADLChart(),
+                    ),
+                  ],
+                ),
+              ),
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.fromLTRB(10, 0, 10, 10),
+            child: Container(
+              child: Card(
+                child: Column(
+                  crossAxisAlignment: CrossAxisAlignment.stretch,
+                  children: <Widget>[
+                    Container(
+                      margin: EdgeInsets.all(10),
+                      child: Text(
+                        'ผลการประเมินการปฏิบัติกิจวัตรประจำวัน',
+                        style:
+                            TextStyle(fontSize: 18, color: Color(0xFFC37447)),
+                      ),
+                    ),
+                    Padding(
+                      padding: const EdgeInsets.fromLTRB(50, 0, 50, 20),
+                      child: PieChart(),
+                    ),
+                  ],
+                ),
+              ),
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+
+  Widget _postopHospitaldashboard(BuildContext context) {
```

Comment:
Consider using camelCase 

_postOpHospitalDashboard

Reply:
Resolve in 143d0ec